tests/unit/BenCorrection.spec.ts

```diff
@@ -156,6 +159,16 @@ describe('Benefit Company Correction component', () => {
     expect(wrapper.findComponent(BenCorrection).exists()).toBe(true)
   })
 
+  it('loads each component', async () => {
+    expect(wrapper.findComponent(YourCompany).exists()).toBe(true)
+    expect(wrapper.findComponent(PeopleAndRoles).exists()).toBe(true)
+    expect(wrapper.findComponent(ShareStructures).exists()).toBe(true)
+    expect(wrapper.findComponent(Articles).exists()).toBe(true)
+    expect(wrapper.findComponent(Detail).exists()).toBe(true)
+    expect(wrapper.findComponent(CertifySection).exists()).toBe(true)
+    expect(wrapper.findComponent(StaffPayment).exists()).toBe(true)
+  })
```

Comment:
Thanks for adding these!

Reply:
That's the least I can do for this unit test.